apps/openmw/mwworld/projectilemanager.cpp

```diff
@@ -136,7 +138,7 @@ namespace MWWorld
     };
 
 
-    void ProjectileManager::createModel(State &state, const std::string &model, const osg::Vec3f& pos, const osg::Quat& orient, bool rotate, std::string texture)
+    void ProjectileManager::createModel(State &state, const std::string &model, const osg::Vec3f& pos, const osg::Quat& orient, bool rotate, bool isMagic, std::string texture)
```

Comment:
isMagic should be renamed to createLight to reflect the variable's real purpose.